docs/DeveloperGuide.adoc

```diff
@@ -873,10 +1004,207 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. Hallper shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Publish calendar
+
+*MSS*
+
+1.
```

Comment:
To be completed.

Reply:
Missing Calendar Use Case : 
-------------------------------
[discrete]
=== Use case: publish calendars

*MSS*

1. JCRC member requests to publish calendar
2. Hallper requests for the intended calendar month
3. JCRC member enters the month
4. Hallper saves an output file in the computer

Use case ends.

[discrete]
=== Use case: Add event on Calendar

*MSS*

1. JCRC member requests to add new event to calendar
2. Hallper requests for all details of event (Which month, date, start time, end time, event name)
3. JCRC member enters all details
4. Hallper creates a new event on the calendar

Use case ends.

*Extensions*

[none]
* 3a. Another event clashes with the same timing

Use case ends.

[discrete]
=== Use case: Delete event on Calendar

*MSS*

1. JCRC member requests to delete an event on the calendar
2. Hallper requests for all details of event (Which month, date, event name)
3. JCRC member enters all details
4. Hallper deletes the specified event on the calendar

Use case ends.

*Extensions*

[none]
* 3a. No such event exist

Use case ends.

Reply:
Code added.